packages/axes/src/compute.js

```diff
@@ -184,7 +184,7 @@ export const computeCartesianTicks = ({
     }
 
     const ticks = values.map(value => ({
-        key: value,
+        key: typeof value === 'number' || typeof value === 'string' ? value : `${value}`,
```

Comment:
This could also be just:
```suggestion
        key: `${value}`,
```